src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Models/RoleGrant.cs

```diff
@@ -0,0 +1,19 @@
+namespace Altinn.Platform.Authorization.Models
+{
+    /// <summary>
+    /// This model describes a role which a particular action is given to.
+    /// example: OrderedMap { "RoleTypeCode": "DAGL", "IsDelegable": true }
```

Comment:
Is it supposed to be orderedMap herer or rather: 

```suggestion
    /// example: RoleGrant { "RoleTypeCode": "DAGL", "IsDelegable": true }
```

Reply:
Yep, it's supposed to be RoleGrant here.